Models/BusinessLogic.cs

```diff
@@ -361,24 +361,33 @@ public async Task<List<ActiveQuiz>> GetActiveQuizzesByActiveQuizIds(List<long> a
         var questions = await database.GetQuizQuestionsByActiveQuizId(activeQuizId);
         var responses = await database.GetRepsonsesByActiveQuizId(activeQuizId);
 
+        //Create a dictionary to keep track of each student and their respective score
         Dictionary<Guid, int> studentsScores = new Dictionary<Guid, int>();
+        //Go through all of the reponses
         foreach (var response in responses) {
+            //If the student doesn't exist yet in the dictionary, add them to it with a score of 0
             if (!studentsScores.ContainsKey(response.UserId)) {
                 studentsScores.Add(response.UserId, 0);
             }
+            //Get the questions that the response if for
             Question question = questions.FirstOrDefault(q => q.QuestionNo == response.QuestionNo);
             if (question != null) {
+                //If the question is a fill blank, check to see if their answer is one of the acceptable answers
                 if (question.AcceptableAnswers != null && question.AcceptableAnswers.Contains(response.FillBlankResponse)) {
```

Comment:
What do you think about doing something like:
question.QuestionType == QuestionType.FillBlank
instead of checking if question.AcceptableAnswers != null or question.MultipleChoiceCorrectAnswers != null